airsonic-main/src/main/java/org/airsonic/player/spring/SonosConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package org.airsonic.player.spring;
+
+import org.airsonic.player.service.SonosService;
+import org.airsonic.player.service.sonos.SonosFaultInterceptor;
+import org.apache.cxf.jaxws.EndpointImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ImportResource;
+
+import javax.xml.ws.Endpoint;
+
+import java.util.Collections;
+
+@Configuration
+@ImportResource({"classpath:META-INF/cxf/cxf.xml", "classpath:META-INF/cxf/cxf-servlet.xml"})
```

Comment:
Since we're removing XML, shouldn't the Sonos configuration also be part of it?

Reply:
Haven't dwelved too much into this, but this config is not coming from airsonic-main. It's probably from another jar or autogenerated. So it isn't part of this PR.